libs/blocks/gnav/gnav.css

```diff
@@ -627,7 +627,13 @@ nav.gnav .gnav-navitem-menu .gnav-promo.card img {
   right: 0;
 }
 
-.gnav-navitem.has-menu.is-open .gnav-navitem-menu > div > div > ul {
+.gnav-navitem:not(.large-menu) .gnav-navitem-menu {
+  box-shadow: 0px 3px 3px 0px rgb(0 0 0 / 20%);
```

Comment:
This isn't being flagged by eslint? I'm pretty sure it doesn't want units on zero

Reply:
Sorry, I meant stylelint